src/double_integrator/visualizer.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
can you add visualizer.py to the CMakeList.txt like this:
https://github.com/RussTedrake/underactuated/blob/master/src/quadrotor2d/CMakeLists.txt#L2

(i think it will then ask you to change a little of the formatting)

Reply:
Done (no format changes requested, apparently)